framework/test/unit/components/storage/entities/round.js

```diff
@@ -271,11 +268,11 @@ describe('Round', () => {
 
 	describe('isPersisted()', () => {
 		beforeEach(async () => {
-			await RoundEntity.create(validRound);
+			return RoundEntity.create(validRound);
 		});
 
 		afterEach(async () => {
-			await RoundEntity.delete(validFilter);
+			return RoundEntity.delete(validFilter);
```

Comment:
Some how this change makes me feel that issue was with the use of `await` rather just the `@sequential` tag. 

Please revert that change and we are good to go with it.